### ISSUE TITLE
Host access Documentation Problem

### ISSUE THREAD

**Issue by bradurani** [kubernetes/minikube#10204]
Hi, the procedure outlined for connecting to the host in the documentation at: https://minikube.sigs.k8s.io/docs/handbook/host-access/
does not work because neither `ping` nor `telnet` are installed on the minikube docker image
```
docker@minikube:~$ ping host.minikube.internal
-bash: ping: command not found
docker@minikube:~$ telnet host.minikube.internal
-bash: telnet: command not found
```

**Comment by priyawadhwa** [comment]
Hey @bradurani thanks for opening this issue. We can fix this by installing `iputils-ping` in our docker base image here:

https://github.com/kubernetes/minikube/blob/master/deploy/kicbase/Dockerfile#L96

If you, or anyone else is interested in adding this and submitting a PR, please feel free to comment /assign on this issue

**Comment by bradurani** [comment]
PR opened: https://github.com/kubernetes/minikube/pull/10262

**Comment by spowelljr** [comment]
@bradurani You seem to have solved this issue with your PR #10262 so I'm going to close this issue, thanks for your contribution!
